src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -129,4 +129,6 @@
     public static final String NOTE_SEQUENCE_NAME = "KRNS_NTE_S";
 
     public static final Set<String> OBJECTS_WITH_IMMEDIATE_BO_LEVEL_NOTE_UPDATE = Collections.singleton("org.kuali.kfs.coa.businessobject.Account");
+    
+    public static final String DATE_FORMAT_MMDDYYYY = "MM/dd/yyyy";
```

Comment:
There is a constant for this in base code: KFSConstants.MONTH_DAY_YEAR_DATE_FORMAT.